visallo/visallo
# Changed: /vertex/edges to use Vertexium vertex query for performance

Before this commit the vertex was retrieved and all edges were evaluated to
determine if they should be included in the response. This change switches to
using a vertex query which will use ElasticSearch under the covers to quickly
find and page results.

This change will also facilitate sorting by some edge property in the future
instead of key order.

Testing Instructions:
- find a vertex with many edges and open the element inspector and expand the edges. This should be much faster for large number of edges

CHANGELOG
Changed: /vertex/edges to use Vertexium vertex query for performance
